deploy/kubernetes/docker/build.sh

```diff
@@ -72,9 +79,10 @@ fi
 
 HADOOP_FILE=hadoop-${HADOOP_VERSION}.tar.gz
 HADOOP_URL=https://archive.apache.org/dist/hadoop/core/hadoop-${HADOOP_VERSION}/${HADOOP_FILE}
-echo "HADOOP_URL is $HADOOP_URL"
+LATEST_HADOOP_URL=${APACHE_MIRROR}/hadoop/core/hadoop-${HADOOP_VERSION}/${HADOOP_FILE}
```

Comment:
Maybe `MIRROR_HADOOP_URL`?

Reply:
OK, I got it after looking into the issue, maybe call this `HADOOP_URL`, and change line 81 to `ARCHIVE_HADOOP_URL`